makefile

```diff
@@ -56,15 +56,14 @@ lintArgs += --vendored-linters --enable-gc
 #   gotype produces false positives because it reads .a files which
 #   are rarely up to date.
 lintArgs += --disable="gotype" --disable="gas" --disable="gocyclo" --disable="maligned"
-lintArgs += --disable="golint" --disable="goconst" --disable="dupl"
+lintArgs += --disable="golint" --disable="goconst" --disable="dupl" --disable="megacheck"
```

Comment:
Whoa, we're disabling megacheck globally here?

Reply:
Yeah, I'm going to fold megacheck back in next in EVG-2803.